webserver/views/user.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 import webserver
 import db.user
+import ujson
```

Comment:
I was unaware of that.
I actually saw it being used somewhere in the code, and when i read about it i thought it would be good to use it.

> UltraJSON is an ultra fast JSON encoder and decoder written in pure C wit...

I'll check and convert all instances to use `json` instead.